scripts/intfutil

```diff
@@ -478,7 +491,7 @@ class IntfStatus(object):
                                 appl_db_port_status_get(self.db, key, PORT_LANES_STATUS),
                                 port_oper_speed_get(self.db, key),
                                 appl_db_port_status_get(self.db, key, PORT_MTU_STATUS),
-                                appl_db_port_status_get(self.db, key, PORT_FEC),
+                                port_oper_fec_get(self.db, key),
```

Comment:
@dgsudharsan as discussed, lets not change this behavior. Existing 'FEC' column would continue to show whatever is configured in CONFIG_DB (i.e same as in APPL_DB).